dockernet/ts-tests/test/main.test.ts

```diff
@@ -0,0 +1,180 @@
+import { Secp256k1HdWallet } from "@cosmjs/amino";
+import { GasPrice, SigningStargateClient } from "@cosmjs/stargate";
+import { fromSeconds } from "@cosmjs/tendermint-rpc";
+import {
+  coinFromString,
+  convertBech32Prefix,
+  decToString,
+  sleep,
+  StrideClient,
+} from "stridejs";
+import { beforeAll, describe, expect, test } from "vitest";
+
+const RPC_ENDPOINT = "http://localhost:26657";
+
+let accounts: {
+  user: StrideClient; // just a normal user account loaded with 100 STRD
+  admin: StrideClient; // the stride admin account loaded with 1000 STRD
+  val1: StrideClient;
+  val2: StrideClient;
+  val3: StrideClient;
+};
+
+// init accounts and wait for chain to start
+beforeAll(async () => {
+  console.log("setting up accounts...");
+
+  const mnemonics: {
+    name: "user" | "admin" | "val1" | "val2" | "val3";
+    mnemonic: string;
+  }[] = [
+    {
+      name: "user",
+      mnemonic:
+        "brief play describe burden half aim soccer carbon hope wait output play vacuum joke energy crucial output mimic cruise brother document rail anger leaf",
+    },
+    {
+      name: "admin",
+      mnemonic:
+        "tone cause tribe this switch near host damage idle fragile antique tail soda alien depth write wool they rapid unfold body scan pledge soft",
+    },
+    {
+      name: "val1",
+      mnemonic:
+        "close soup mirror crew erode defy knock trigger gather eyebrow tent farm gym gloom base lemon sleep weekend rich forget diagram hurt prize fly",
+    },
+    {
+      name: "val2",
+      mnemonic:
+        "turkey miss hurry unable embark hospital kangaroo nuclear outside term toy fall buffalo book opinion such moral meadow wing olive camp sad metal banner",
+    },
+    {
+      name: "val3",
+      mnemonic:
+        "tenant neck ask season exist hill churn rice convince shock modify evidence armor track army street stay light program harvest now settle feed wheat",
+    },
+  ];
+
+  // @ts-expect-error
```

Comment:
whats the `// @ts-expect-error` for?

Reply:
`accounts` is defined as:
```ts
let accounts: {
    user: StrideClient;
    admin: StrideClient;
    val1: StrideClient;
    val2: StrideClient;
    val3: StrideClient;
}
```
and here we initialize it with all of its fields as `undefined`, which is a type error. to make this error go away with can either intialize all the fields with dummy values, mark this like as `// @ts-expect-error` or define `accounts` as:
```ts
let accounts: {
    user?: StrideClient;
    admin?: StrideClient;
    val1?: StrideClient;
    val2?: StrideClient;
    val3?: StrideClient;
}
```
the problem with defining it like that is that whenever we'd want to access one of the fields we'd have to do `account?.user` or check if it's not `undefined`, which is very annoying.

this is just an initialization code so I feel comfortable to tell typescript to shut up.

Reply:
makes sense, good call!